cmd/mattermost/commands/webhook_test.go

```diff
@@ -210,3 +210,54 @@ func TestCreateOutgoingWebhook(t *testing.T) {
 		t.Fatal("Failed to create incoming webhook")
 	}
 }
+
+func TestDeleteWebhooks(t *testing.T) {
+	th := api4.Setup().InitBasic().InitSystemAdmin()
+	defer th.TearDown()
+	adminClient := th.SystemAdminClient
+
+	th.App.UpdateConfig(func(cfg *model.Config) { cfg.ServiceSettings.EnableIncomingWebhooks = true })
+	th.App.UpdateConfig(func(cfg *model.Config) { cfg.ServiceSettings.EnableOutgoingWebhooks = true })
+	th.App.UpdateConfig(func(cfg *model.Config) { cfg.ServiceSettings.EnablePostUsernameOverride = true })
+	th.App.UpdateConfig(func(cfg *model.Config) { cfg.ServiceSettings.EnablePostIconOverride = true })
+
+	defaultRolePermissions := th.SaveDefaultRolePermissions()
```

Comment:
It would be way easier using directly the app layer instead of the Api layer. If you use the app layer, you don't need to change the permissions to create anything.

Reply:
All the other tests in `webhook_test.go` were using Api layer, thats why I also continued it. Do you still want to change it?

Reply:
I think so, is a better solution. Actually the CLI doesn't use the API for do the work, I wont expect to use it for the tests.

Probably a sensible next step would be change the rest of the tests to use the "App" layer for all the other tests.